lib/osml/model_runner/mr_dataplane.ts

```diff
@@ -444,15 +444,72 @@ export class MRDataplane extends Construct {
       cluster: this.cluster,
       minHealthyPercent: 100,
       securityGroups: this.securityGroups,
-      vpcSubnets: props.osmlVpc.selectedSubnets
+      vpcSubnets: props.osmlVpc.selectedSubnets,
+      desiredCount: this.mrDataplaneConfig.MR_DEFAULT_DESIRE_COUNT
     });
 
+    interface LoggingOptions {
```

Comment:
I feel like this interface should be defined at the top of the file in the entire fluent bit/logging stuff should be migrated into it's own component... but we can spin up a task for that later

Reply:
Updated in Rev2. Please take a look.